scripts/hail_batch/vep/run_vep.py

```diff
@@ -25,6 +25,7 @@ def main(mt: str):
     mt = mt.filter_rows(hl.len(mt.alleles) == 2)
     mt = mt.filter_rows(mt.alleles[1] != '*')
     vep = hl.vep(mt, config='file:///vep_data/vep-gcloud.json')
+    vep = vep.rows()
     vep_path = output_path('vep105_GRCh38.mt')
```

Comment:
Should we use VEP 104 here, as I thought we decided to use GENCODE 38 for TOB-WGS?

Reply:
I think this would need to be rebuilt, right? https://centrepopgen.slack.com/archives/C018KFBCR1C/p1661486090993449?thread_ts=1658208231.938539&cid=C018KFBCR1C

Reply:
Yes, I believe so.